pages/api/fipe/veiculos/v1/[vehicleType].js

```diff
@@ -0,0 +1,29 @@
+import app from '@/app';
+import BadRequestError from '@/errors/BadRequestError';
+
+import {
+  listCarByMaker,
+  listMotorcycleByMaker,
+  listTruckByMaker,
+} from '@/services/fipe/vehiclesByMakers';
+
+const VEHICLE_TYPES = {
+  caminhoes: listTruckByMaker,
+  carros: listCarByMaker,
+  motos: listMotorcycleByMaker,
+};
+
+async function FipeVehicles(request, response) {
+  const makerCode = request.query.codigoMarca;
```

Comment:
Além disso, por convenção, utilizamos snake case para as interfaces das apis
```suggestion
  const makerCode = request.query.codigo_marca;
```

Reply:
code style atualizado.